core/src/main/java/org/apache/calcite/sql/SqlRangeNStartEnd.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql;
+
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+import java.util.List;
+
+public class SqlRangeNStartEnd extends SqlCall{
+
+  final SqlNode startLiteral;
```

Comment:
Let's make this (and other fields in this class) public.

Reply:
Fxied in https://github.com/googleinterns/calcite/pull/212/commits/d6bf9941b266f50cde7f6b0311d9580d7e98684b